recipes/r-metadig/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "r-metadig" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NCEAS/metadig-r/archive/refs/tags/v0.2.1.tar.gz
```

Comment:
The Conda Forge R team prefers to work downstream of CRAN. Is there a reason why this cannot be on CRAN?

Reply:
No particular reason, I just needed this package to be on Conda to be used in Galaxy and I didn't think of releasing it to CRAN

Reply:
But I can try to use bioconda thanks if you think it would be more appropriate